lib/src/middleware/load_file.dart

```diff
@@ -1,16 +1,29 @@
 import 'dart:html';
 
+import 'package:quiver/async.dart';
 import 'package:redux/redux.dart';
+import 'package:scadnano/src/actions/actions.dart';
 
 import '../actions/actions.dart' as actions;
 import '../app.dart';
 import '../state/app_state.dart';
 import '../util.dart' as util;
 
 load_file_middleware(Store<AppState> store, action, NextDispatcher next) {
-  next(action);
+  if (action is actions.PrepareToLoadDNAFile && !action.unit_testing) {
```

Comment:
Add some comments here acknowledging that this is hacky and that we don't know why it works. Also enumerate the things we tried that didn't work. In the future I'm likely to get the idea that one of those ideas should work, and I'd like to be reminded that we already tried it.